apps/studio/src/server/modules/resource/resource.types.ts

```diff
@@ -1,5 +1,9 @@
-export interface NavbarItem {
-  name: string
+import { type IsomerSiteProps } from '@opengovsg/isomer-components'
+
+export type Navbar = { items: IsomerSiteProps['navBarItems'] }
+
+export interface FooterItem {
```

Comment:
nit: I suspect most of the types that we need on the CMS should be able to be derived from the exported props one way or another, so we might not need to re-define them here.

Reply:
exported for ease of use cos it's very tedious to type `Isomer...['...']` every time ._.